clouddriver-cloudfoundry/src/main/groovy/com/netflix/spinnaker/clouddriver/cloudfoundry/CloudFoundryCloudProvider.groovy

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2018 Pivotal Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.cloudfoundry
+
```

Comment:
We try to write all new code in [Java/Kotlin](https://www.spinnaker.io/community/contributing/back-end-code/) -- take for example the new [k8s provider in clouddriver](https://github.com/spinnaker/clouddriver/tree/master/clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2)

It's not exactly a hard requirement, but would be nice

Reply:
Oh boy. I wish we had known this before we wrote the whole provider.

Reply:
There are some notes on the site here (https://www.spinnaker.io/community/contributing/back-end-code/), but we could definitely do a better job of publicizing this.